qgis-samgeo-plugin/_samgeo_lib.py

```
@@ -0,0 +1,176 @@
+    _ensure_venv_on_path()
+
+    plugin_dir = Path(__file__).resolve().parent
+
+    # Normal import first (works when plugin package isn't named `samgeo`)
+    try:
+        imported = importlib.import_module("samgeo")
+        if not _is_module_from_dir(imported, plugin_dir):
+            _CACHED = imported
+            return imported
+    except Exception:  # nosec B110
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)